### DISCUSSION TITLE
Responsive behavior within Gmail split pane view

### DISCUSSION THREAD

**Discussion by ns-alexmueller** [mjmlio/mjml#2683] [Q&A] [answered]
Hi folks,

Gmail allows users to [view emails within a split pane](https://support.google.com/mail/answer/9499937?hl=en). It looks like the my mjml emails' responsive behaviors are being dictated by the browser width rather than the pane width. For example, my email uses a breakpoint of 834px to switch between mobile and desktop styles. If the browser window is >834px wide, then the email will be rendered with the desktop styling even if the pane is <834px. 

Does anyone know a way around this? Or is this just a quirk of gmail?

Thanks!

**Comment by iRyusa** [comment, marked as answer]
Nothing we can about this, media queries rely on browser width and
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries are
not available in email.

**Reply by ns-alexmueller** [reply]
got it. thanks for the quick response!
